math/check_perfect_square.cpp

```diff
@@ -0,0 +1,88 @@
+/**
+ * @file
+ * @brief This program aims at checking that the given number is perfect square or not
+ *
+ */
+
+#include <cassert>
+#include <iostream>
+
+/** Getting the integral square root of the number using binary search **/
```

Comment:
Missing parameter and return statement documentation.
This is how it should be documented (same in other parts of the code):

```suggestion
/**
 * @brief Getting the integral square root of the number using binary search
 * @param num description
 * @returns description
 */
```